src/libpsl-native/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-z,relro,-z,now")
 elseif (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl")
+elseif (${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl")
```

Comment:
It seems strange to pass `-Wl` with no arg. Of course the oddity was just duplicated from the Darwin case (where it exists in the first public version of this file).

It seems to me we ought to just mimic the Linux behaviour here, e.g. make the existing case above `if (${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")`

Reply:
I found it unusual too, but opted to include it as I assumed if it was important enough for `"Darwin"` to have it then so should `"FreeBSD"`. I will go ahead and add `"FreeBSD"` to the `"Linux"` case as suggested. Thanks!